web/themes/custom/subtheme_videogame/src/sass/layout/_page.content.scss

```diff
@@ -3,3 +3,28 @@
 .page__content {
 
 }
+
+.layout-builder__section {
+  color: #0f0f0f;
+}
+
+div.layout:nth-child(2) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) {
```

Comment:
What do you want to target with this long and quite specific selector?

Reply:
It is the 'product name' block which is in the same section with the image we use for the game and thanks to the position property it is displayed on the image as is on the wireframe.

Reply:
I believe there are classes added to these section by default, so can you use more meaningful classes for theming? Also, the styles in this file are not restricted to the product page, if you create a new content type with layout builder then the same styles will apply, which we don't want.
Let's see an example:
The page content's class is `node-type--product`, the title has `field--title` and the image has `field--field-image` class. In this case you can write something like this (as you're writing SCSS now, you can use nesting and concatenation):
```scss
.node-type--product {
  .field {
    &--title {
      // Title styles come here.
    }
    &--field-image {
      // Image styles come here.
    }
  }
}
```